src/index.scss

```diff
@@ -0,0 +1,7 @@
+
+$mdc-theme-primary      : #03A9F4; // Light blue 500
+$mdc-theme-primary-light: #B3E5FC; // Light blue 100
+$mdc-theme-secondary    : #69F0AE; // Green A200
+$mdc-theme-background   : #FFFFFF; // White
+
+@import "material-components-web/material-components-web";
```

Comment:
woah what this

Reply:
SASS bruh. It's how MDC-web gives you control over your [color theming](https://material.io/components/web/docs/theming/).